src/test/java/org/springframework/data/couchbase/repository/ReactiveCouchbaseRepositoryQueryIntegrationTests.java

```diff
@@ -72,7 +76,7 @@ void shouldSaveAndFindAll() {
 		Airport vie = null;
 		Airport jfk = null;
 		try {
-			vie = new Airport("airports::vie", "vie", "loww");
+			vie = new Airport("airports::vie", "vie", "low1");
```

Comment:
After running tests, I found some "vie", "loww" Airport records. So I made them unique to figure out what test was creating them and not deleting them.